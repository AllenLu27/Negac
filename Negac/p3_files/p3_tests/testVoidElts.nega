int fn(void a){
	void b;
	void c;
}
