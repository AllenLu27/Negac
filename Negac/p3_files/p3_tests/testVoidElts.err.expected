FATAL [1,13]: Invalid type in declaration
FATAL [2,7]: Invalid type in declaration
FATAL [3,7]: Invalid type in declaration
Semantic analysis failed
